Leaf API.postman_collection.json

```diff
@@ -1174,6 +1174,161 @@
 										}
 									]
 								},
+								{
+									"name": "Stara",
+									"item": [
+										{
+											"name": "Get Stara credentials of the LeafUser",
+											"event": [
+												{
+													"listen": "test",
+													"script": {
+														"exec": [
+															"pm.test(\"Successful GET request\", function () {",
+															"    pm.expect(pm.response.code).to.be.oneOf([200,404]);",
+															"});"
+														],
+														"type": "text/javascript"
+													}
+												}
+											],
+											"request": {
+												"method": "GET",
+												"header": [
+													{
+														"key": "Authorization",
+														"value": "Bearer {{leaf_token}}",
+														"type": "text"
+													},
+													{
+														"key": "Content-Type",
+														"value": "application/json",
+														"type": "text"
+													}
+												],
+												"url": {
+													"raw": "https://{{leaf_api_url}}/services/usermanagement/api/users/{{leaf_user_id}}/stara-credentials",
+													"protocol": "https",
+													"host": [
+														"{{leaf_api_url}}"
+													],
+													"path": [
+														"services",
+														"usermanagement",
+														"api",
+														"users",
+														"{{leaf_user_id}}",
+														"stara-credentials"
+													]
+												}
+											},
+											"response": []
+										},
+										{
+											"name": "Add Stara credentials to the LeafUser",
+											"event": [
+												{
+													"listen": "test",
+													"script": {
+														"exec": [
+															"pm.test(\"Successful POST request\", function () {",
+															"    pm.expect(pm.response.code).to.be.oneOf([201]);",
+															"});"
+														],
+														"type": "text/javascript"
+													}
+												}
+											],
+											"request": {
+												"method": "POST",
+												"header": [
+													{
+														"key": "Authorization",
+														"value": "Bearer {{leaf_token}}",
+														"type": "text"
+													},
+													{
+														"key": "Content-Type",
+														"value": "application/json",
+														"type": "text"
+													}
+												],
+												"body": {
+													"mode": "raw",
+													"raw": "{\n    \"apiKey\": \"{{stara_api_key}}\",\n    \"accessToken\": \"{{stara_access_token}}\",\n    \"accessTokenClient\": \"{{stara_access_token_client}}\",\n    \"refreshToken\": \"{{stara_refresh_token}}\"\n}",
```

Comment:
We need to update the postman collection to include the new variables and also explain what they are :smile:

Reply:
c826cb7d8b7d1656098c1cb5c5b0fa67594a5180
I think the names are pretty descriptive, do you think it needs more explanation?

Reply:
I suppose the explanation of variables is something like this:

![image](https://user-images.githubusercontent.com/9080762/148591342-962e7951-440c-425d-83e7-f16a307f53de.png)

Is this right?